Add displayBanner and removeBanner methods (#38)
wazo-communication/euc-plugins-js-sdk: documentation/docs/plugins/sdk.md

When a plugin is unloaded because of a user action (ex: logout), this listener `app.onPluginUnLoaded()` will be called. Other listeners will be disabled when unloaded. To prevent side effects, it's important to remove any watcher added in the initializing phase.

> When the user logs in again, the `backgroundScript` will be loaded again, please make sure to release everything in `onPluginUnLoaded`.

Reviewer: ```suggestion
When the user logs in again, the `backgroundScript` will be reloaded, please make sure to release everything in `onPluginUnLoaded`.
```

We can display a modal in the `backgroundScript` with:

```js
app.displayModal({ url, title, text, htmlText, height, width, hideCloseButton });
```

If `url` is present, the modal will display an iframe with the content of the url.
If `htmlText` is present, the modal will display this text in a html contact, otherwise the `text` attribute will be used.
The `height` and `width` accept valid CSS values, like `500px` or `80%`.

> `hideCloseButton` (default to false), indicate if the close button should be displayed of it the user should handle the closing of the modal in the html content (throught `app.removeModal()` method).

Reviewer: ```suggestion
`hideCloseButton` (default to false) determines if the close button should be displayed, or if the user should handle closing of the modal in the html content (through `app.removeModal()` method).
```

:::note

> On mobile, links with a `taget="_blank"` attributes will be opened in the user's default browser.

Reviewer: ```suggestion
On mobile, links with a `target="_blank"` attribute will be opened in the user's default browser.
```

};
```

#### Displaying a banner (mobile only)

We can display a banner in the `backgroundScript` with:

```js
app.displayBanner({ url, height, width, hideCloseButton });
```

If `url` is present, the modal will display an iframe with the content of the url.
The `height` accepts valid CSS values, like `500px` or `80%`.
The `width` accepts valid CSS values, like `500px` or `80%`, used on WDA (min: `300px`).

> `hideCloseButton` (default to false), indicate if the close button should be displayed of it the user should handle the closing of the modal in the html content (throught `app.removeBanner()` method).

Reviewer: ```suggestion
`hideCloseButton` (default to false) determines if the close button should be displayed, or if the user should handle closing of the modal in the html content (using `app.removeBanner()` method).
```

> We could then call `app.removeBanner()` in the `backgroundScript` or the loaded content.

Reviewer: ```suggestion
We can then call `app.removeBanner()` in the `backgroundScript` or the loaded content.
```

:::note

> On mobile, links with a `taget="_blank"` attributes will be opened in the user's default browser.

Reviewer: ```suggestion
On mobile, links with a `target="_blank"` attribute will be opened in the user's default browser.
```

:::

:::note

> On WDA the banner will be integrated with other banners like incoming calls. Other banners will be displayed below.

Reviewer: ```suggestion
On WDA, the banner will be integrated with other banners like incoming calls. Other banners will be displayed below.
```
